openhtf/exe/__init__.py

```diff
@@ -128,11 +127,11 @@ def _RunTestLoop(self):
       exit_stack.callback(lambda: setattr(self, '_current_exit_stack', None))
 
       plug_manager = self._MakePlugManager(suppressor)
-      phase_executor = self._MakePhaseExecutor()
+      executor = self._MakePhaseExecutor()
```

Comment:
i agree, this made me sad, but phase_executor is already taken by outer scope, and i wasn't feeling terribly creative... suggestions?